concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -279,7 +279,7 @@ resources:
     type: git
     source:
       uri: https://github.com/alphagov/paas-billing
-      tag_filter: v0.15.0
+      tag_filter: v0.17.0
```

Comment:
Did you mean to include this in this PR? I don't see how it's related to this (or #1271).

Reply:
It got in somehow when I rebased this branch from Hector's PR. A master rebase solved the issue - but this is still WIP.